### PR TITLE
Added price history

### PR DESCRIPTION
**Pull request by glenviewjeff**
Added price history.  Should probably test with other URLs/houses as I only tried with one.  FYI I was lazy and didn't put any null checks in.

### CODE REVIEW

**Comment by hahnicity** [comment]
Looks pretty good if you've tested everything. There are some indentation inconsistencies I noticed along with some coding standard differences: Main thing I can say is tabs and whitespace. If you can fix that up the [pep8 documentation](https://www.python.org/dev/peps/pep-0008/#code-lay-out) has pretty good guidelines on how to do this for python. I'll merge your code after everything is fixed.

**Comment by glenviewjeff** [comment]
I think I fixed the couple of indentation/tab errors?  I'm using a crappy-ass text editor (notepad++)  instead of an IDE.  I didn't notice any other tabs, and being a Python newbie it's hard for me to notice style errors.  If you could correct any remaining whitespace issues I'd be grateful.  Either way there's now tax and price history.

File: scrapezillow/scraper.py [hunk 1]
```diff
@@ -117,7 +117,57 @@ def validate_scraper_input(url, zpid):
...
 
-
+def _get_ajax_url(soup, label):
+    javascripts = soup.findAll("script",type="text/javascript")
+    for javascript in javascripts:
+        if javascript.string and javascript.string.find(label):
+            pattern = r'ajaxURL:"([\w\s/~&=\-\?\^\+\.]*)",jsModule:"' + label +'"'
+            url = re.search(pattern,'divId:"' + javascript.string)
+            if url:
+                full_url = "http://www.zillow.com" + url.group(1)
```
Comment (hahnicity) [inline scrapezillow/scraper.py]: thanks for fixing the tabbing

**Comment by hahnicity** [comment]
I'd suggest using vim or emacs if you're going to be editing files on linux. Otherwise if you use windows I think PyCharm should be useful. A coworker of mine also uses Sublime to great effect.

**Comment by hahnicity** [comment]
I just messed around with your patch and it looks excellent! Thanks for your contribution!

**Comment by glenviewjeff** [comment]
Glad you liked.  Later I'll add Google scraping so you can use address based lookups and probably Zestimate history lookup by date.  Both of those should be pretty straightforward.  I'll need both of those to collect data for my project.  

As far as editors go, I'd normally use a fully code-aware IDE (Eclipse) but if I had to pick between lesser editors, I'd stick with notepad++.  I've spent far more than my fair share of time w/vi/emacs. :)

Jeff

**Pull request merged**